cordapp-cpk/src/test/resources/external-cordapp/build.gradle

```diff
@@ -0,0 +1,61 @@
+plugins {
+    id 'net.corda.plugins.cordapp-cpk'
+}
+
+evaluationDependsOnChildren()
```

Comment:
We don't need `evaluationDependsOnChildren()` if we always apply `java-library` in `allprojects {}`. (Because `evaluationDependsOnChildren()` is one of those things that you never want in a Gradle project if you can help it, just like `afterEvaluate {}`.)